Bottom: 52e815ff29403e919a8278c5c0ff7456fe06421b
Top:    6dcb9d94de72979053d2c791036aeb2ca004a371
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 17:58:48 +0000

hid/common: Don't leak the dir resource in hid_load_dir()

Add missing closedir() call.

(Caught with cppcheck)


---

diff --git a/src/hid/common/hidinit.c b/src/hid/common/hidinit.c
index 0bdd395..3695f7d 100644
--- a/src/hid/common/hidinit.c
+++ b/src/hid/common/hidinit.c
@@ -117,6 +117,7 @@ hid_load_dir (char *dirname)
       free (path);
     }
   free (dirname);
+  closedir (dir);
 }
 
 void
